### PR TITLE
Fixes #8239: Inventory messes up Last logged in user date when last user was a reboot

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/8239

### CODE REVIEW

File: rudder-agent/SOURCES/patches/fusioninventory/0015-ignore-last-login-if-reboot.patch [hunk 1]
```diff
@@ -0,0 +1,12 @@
+diff -upr fusioninventory-agent/lib/FusionInventory/Agent/Task/Inventory/Generic/Users.pm fusioninventory-agent-b/lib/FusionInventory/Agent/Task/Inventory/Generic/Users.pm
+--- fusioninventory-agent/lib/FusionInventory/Agent/Task/Inventory/Generic/Users.pm	2014-03-10 12:34:32.000000000 +0100
++++ fusioninventory-agent-b/lib/FusionInventory/Agent/Task/Inventory/Generic/Users.pm	2016-05-09 15:34:36.706050533 +0200
+@@ -152,6 +152,8 @@ sub _getLastUser {
+ 
+     my $last = getFirstLine(%params);
+     return unless $last;
++    # if last line is a reboot, ignore it
++    return if $last =~ /^reboot.*system boot/;
```
Comment (peckpeck) [inline rudder-agent/SOURCES/patches/fusioninventory/0015-ignore-last-login-if-reboot.patch]: Shouldn't we try to get next line if the last one was a reboot ?

**Comment by jooooooon** [comment]
Ping @amousset

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Comment by peckpeck** [comment]
Good, merging

**Pull request merged**